packages/web-components/src/components/ic-search-bar/ic-search-bar.stories.mdx

```diff
@@ -212,6 +212,78 @@ import readme from "./readme.md";
 
 <Canvas>
   <Story name="External filtering" parameters={{ loki: { skip: true } }}>
+    {(args) =>
+      html` <ic-search-bar
+          label="Search for coffee"
+          placeholder="Type to search"
+          debounce="300"
+          disable-filter="true"
+          id="external-filter-search"
+          timeout="1000"
```

Comment:
really minor one, but don't think timeout prop is needed as only used when loading prop set